packages/firestore/src/api/bundle.ts

```diff
@@ -138,3 +119,30 @@ export class LoadBundleTask
     }
   }
 }
+
+export function loadBundle(
```

Comment:
It looks like we still need to add this to firestore-exp, but we can do this as a follow up.

Reply:
Not sure what this means, can you elaborate?

Reply:
It looks like we are at least missing the top-level exports in `exp/index.ts`.